app/controllers/books_controller.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+class BooksController < ApplicationController
+  def index
+    @categories = Category.all
```

Comment:
Imho it should be in application_controller, if im not mistaken we have categories in drop-down menu, don't we?

Reply:
fixed